source/configure/config-proxy-apache2.rst

```diff
@@ -2,38 +2,95 @@
 3. Create the above mentioned configuration file. It is often helpful to start with a copy of ``000-default.conf`` or ``default-ssl.conf`` (on Ubuntu).
 4. Edit your configuration using the guide below:
 
-    1. If you're not setting up a subdomain, your ``ServerName`` will simply be set to ``mydomain.com``.
-    2. ``ServerAlias`` can been added too if you want to capture ``www.mydomain.com``.
+    1. If you're not setting up a subdomain, your ``ServerName`` will simply be set to ``mymattermost.tld``.
+    2. ``ServerAlias`` can been added too if you want to capture any other (sub)domains.
     3. Remember to change the values to match your server's name, etc.
-    4. If you have enabled TLS in the Mattermost settings, you must use the protocol ``wss://`` instead of ``ws://`` in the ``RewriteRule``.
+    4. If you have enabled TLS in the Mattermost settings, you must also enable the SSL module in Apache (usually with ``a2enmod ssl``) and change the ``http://`` part of the proxy destination to ``https://`` additionally to enabling the ``SSLProxyEngine``.
     5. To serve requests on a different port (such as 8443), in addition to setting the port in the VirtualHost element, add ``Listen 8443`` on a separate line before the VirtualHost line.
 
 .. code-block:: apacheconf
 
         <VirtualHost *:80>
-          # If you're not using a subdomain you may need to set a ServerAlias to:
-          # ServerAlias www.mydomain.com
-          ServerName mysubdomain.mydomain.com
-          ServerAdmin hostmaster@mydomain.com
+          ServerName mymattermost.tld
+          # Uncomment the following line if this host also should be reachable on a different domain
+          # ServerAlias sub.mymattermost.tld
+          ServerAdmin hostmaster@mymattermost.tld
+          
+          RewriteEngine On
+          RewriteRule "^/(.*)" "https://mymattermost.tld/$1" [R=301,L]
+        </VirtualHost>
+          
+        <VirtualHost *:443>
+          ServerName mymattermost.tld
+          # Uncomment the following line if this host also should be reachable on a different domain
+          # ServerAlias sub.mymattermost.tld
+          ServerAdmin hostmaster@mymattermost.tld
+          ProxyPreserveHost On
+
+          ProxyPassMatch "^/(api/v[0-9]+/(users/)?websocket)$" "http://127.0.0.1:8065/$1" upgrade=websocket
+          ProxyPassMatch "^/(plugins/focalboard/ws/.*$)" "http://127.0.0.1:8065/$1" upgrade=websocket
+          ProxyPass / http://127.0.0.1:8065/
+          ProxyPassReverse / http://127.0.0.1:8065/
+          ProxyPassReverseCookieDomain 127.0.0.1 mymattermost.tld
+          
+          SSLEngine On
+          SSLCertificateFile /path/to/your/certificate.pem
+          SSLCertificateKeyFile /path/to/your/certificate.key
+          
+          # Uncomment the following line if your Mattermost server is requiring SSL/TLS connections
+          # SSLProxyEngine On
+        </VirtualHost>
+
+6. Restart Apache2.
```

Comment:
Step 6 currently doesn't align to either the initial set of steps nor the substeps, so I'm unclear about which set of steps it applies to. 

If it's truly step 6 and applies to the substeps, please adjust the indentation and update the step from 6 to f.

And if it's actually step 5, it's fine as-is from an indentation perspective, but needs to be renumbered.

Reply:
Fixed, there is no step 6 anymore - it's 5.